app/assets/stylesheets/components/overview.scss

```diff
@@ -1,17 +1,5 @@
 .summary-background {
-  background: $colour-blue;
-}
-
-.summary-link-colour:link {
-  color: $colour-white;
-}
-
-.summary-link-colour:visited {
-  color: $colour-white;
-}
-
-.summary-link-colour:hover {
-  color: $colour-white;
+  border: 3px solid $colour-blue;
```

Comment:
This seems like a significant change visually. Do you think we can make it work like this?
![Kapture 2020-09-17 at 9 55 46](https://user-images.githubusercontent.com/3193694/93448930-0e96c480-f8cc-11ea-942e-59269e825383.gif)

Reply:
think you might be caching old CSS, the new version is this: 

file:///home/steph/Pictures/Screenshot%20from%202020-09-17%2010-25-31.png

Reply:
![Screenshot from 2020-09-17 10-25-31](https://user-images.githubusercontent.com/107635/93452538-6df6d380-f8d0-11ea-96bc-c36eb9aa8b04.png)

Reply:
Yes that's what I meant when I said the change looks significantly different visually. The gif I attached keeps the look how it used to be while still keeping the contrast acceptable.

Having said that, I just went back to the audit doc and the issue was actually having white text over yellow background, when links in those boxes are focused. So the only change we need to make is to change the link text to black when focused. I've checked the contrast against the blue background and it seems acceptable for WCAG 2.1A.